src/cloud/components/DocPage/DocContextMenuActions.tsx

```diff
@@ -127,32 +131,36 @@ export function DocContextMenuActions({
         description: error.message,
       })
     }
-  }, [getUpdatedDoc, settings, pushMessage])
+  }, [subscription, getUpdatedDoc, doc.title, pushMessage])
 
   const exportAsPdf = useCallback(async () => {
+    if (subscription == null) {
+      return
+    }
     const updatedDoc = getUpdatedDoc()
     if (updatedDoc.head == null) {
       return
     }
     try {
-      const previewStyle = defaultPreviewStyle({
-        theme: selectV2Theme(settings['general.theme']),
-      })
       const pdfName = `${filenamifyTitle(updatedDoc.title)}.pdf`
-      const htmlString = await convertMarkdownToPdfExportableHtml(
-        updatedDoc.head.content,
-        settings,
-        previewStyle
-      )
       if (usingElectron) {
```

Comment:
From the paid pdf exporting support, I don't think we need another behavior for exporting pdf and html files. Please make the desktop app also use the same exporting api.

Reply:
Resolved